charts/inbox-agent/templates/operator/deployment.yaml

```diff
@@ -74,16 +74,16 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: INBOX_SERVER_JMAP_ENDPOINT
```

Comment:
Move these envs as args

Reply:
It would be a bit complex to parse the endpoints from flags, since the reverted jmap-client initializes before the flags can be parsed by cobra. I think it's best if we do this after the jmap/webadmin clients are refactored.